examples/basicauth/main.go

```diff
@@ -34,14 +35,14 @@ func main() {
 	}
 
 	client := github.NewClient(tp.Client())
-	user, _, err := client.Users.Get("")
+	user, _, err := client.Users.Get(context.Background(), "")
 
 	// Is this a two-factor auth error? If so, prompt for OTP and try again.
 	if _, ok := err.(*github.TwoFactorAuthError); err != nil && ok {
 		fmt.Print("\nGitHub OTP: ")
 		otp, _ := r.ReadString('\n')
 		tp.OTP = strings.TrimSpace(otp)
-		user, _, err = client.Users.Get("")
+		user, _, err = client.Users.Get(context.Background(), "")
```

Comment:
In examples I think it would be better to use `context.TODO()` tbh to emphasize to the user that they should set as appropriate.